src/commands/purge.ts

```diff
@@ -8,7 +8,7 @@ export default class PurgeCommand implements ICommand {
 	permissionLevel = PermissionLevel.Moderator;
 	guildOnly = false;
 
-	usageText = ";purge <amount>";
+	usageText = ";purge [amount]";
```

Comment:
I think it would be good practice to use `[]` for optional parameters and `<>` for required ones, for this command the amount is required.

This is not yet the rule but we should be clear on what is what for the help command. So this is the time we define those standards within the bot. What is your opinion on this? How should we distinguish between required and optional parameters?

Reply:
I propose we adhere to standards like docopts. We are not using arguments in a very complex way yet, but it's indeed good to think about it now. I think `[]` for optional and `<>` for required is the right choice. For a multiple possible set required values `(option1|option2)` is suitable, I think. So this would make for:
```
;help
;stats
;livereset
;purge <amount>
;config (set|add|remove) <key> <value>
```
This would also be correct according to something like docopts.